src/com/hms/flexyosisoftconnector/DataPoster.java

```diff
@@ -150,6 +150,10 @@ public void run()
 
             if(retval)
             {
+
+               // pop from historical log queue for first tag
+               //((Tag) OSIsoftConfig.tags.get(0)).popNumTagsFromQueue(2);
+

```

Comment:
Is this something that is going to get uncommented and used?

Reply:
missed this when I was deleting comments. should be deleted